rust/rope/docs/MetricsAndBoundaries.md

```diff
@@ -0,0 +1,80 @@
+#Metrics and Boundaries
```

Comment:
missing space after `#`

Reply:
thanks for catching this, should be fixed now